src/parser.rs

```diff
@@ -583,8 +583,11 @@ where
 
         checkpoint
     }
+    fn parse_pipe(&mut self) -> Checkpoint {
+        self.parse_left_assoc(Self::parse_simple, T!["|>"] | T!["<|"])
```

Comment:
shouldn't these have different associativity?

Reply:
You're right - it wasn't explicitly mentioned in the RFC but using nix repl i see that <| is right associative